packages/server/lib/factory/hooksFactory.js

```diff
@@ -164,16 +164,13 @@ module.exports = function hooksFactory({
       try {
         const { context } = event;
```

Comment:
If you change `const { context } = event;` to `const { context, error } = event;`then you could use error.* instead of event.error.*.